Makefile

```diff
@@ -146,6 +149,7 @@ endif
 
 .PHONY: lint
 lint: golangci-lint
+	@echo "Linting vs commit '$(call err_if_empty,EPOCH_TEST_COMMIT)'"
```

Comment:
I don't think `EPOCH_TEST_COMMIT` is actually used here, is it?

Reply:
TBH I'm not sure if it's used by the underlying linting tool or not.  I'm not opposed to removing it.

Reply:
lint acts on a snapshot, i.e. the current source tree. It should not know or care about git history.